packages/client/src/components/sidebar-left.js

```diff
@@ -455,7 +455,7 @@ const component = (state, emit) => {
     : docTitle
 
   return html`<aside class='Sidebar --on-left'>
-    <section class='Sidebar-section'>
+    <section class='Sidebar-section --bottom-line'>
```

Comment:
Aaaa! Check your dark theme!

![image](https://user-images.githubusercontent.com/9429556/36911049-50237e62-1e3a-11e8-8952-2297cf7a495c.png)

Reply:
(I mentioned this above!)

Reply:
Woah, I entirely missed that @towerofnix. Maybe actually submit a review next time? :package: